google/auth/pluggable.py

```diff
@@ -301,11 +304,7 @@ def _parse_subject_token(self, response):
                     response["code"], response["message"]
                 )
             )
-        if "expiration_time" not in response:
-            raise ValueError(
-                "The executable response is missing the expiration_time field."
-            )
-        if response["expiration_time"] < time.time():
+        if "expiration_time" in response and response["expiration_time"] < time.time():
```

Comment:
Let's enforce expiration_time if an output_file was specified. So an exception should be thrown even if calling the executable and the expiration_time is not present if the credential configuration specifies an output_file.